shell-completion/zsh/_networkctl

```diff
@@ -24,10 +26,10 @@
             local -a _links
             cmd="${${_networkctl_cmds[(r)$words[1]:*]%%:*}}"
             case $cmd in
-                (list|status|up|down|lldp|delete|renew|forcerenew|reconfigure)
-                    _links=( "${(foa)$(networkctl list --no-legend | awk 'BEGIN{OFS=":"} {sub(/[[ \t]+/, ""); print $2,$0}' 2>/dev/null)}" )
+                (list|status|up|down|cat|edit|lldp|delete|renew|forcerenew|reconfigure)
+                    for link in ${(f)"$(_call_program links networkctl list --no-legend)"}; do _links+=($link[(w)2]:$link); done
                     if [[ -n "$_links" ]]; then
-                         _describe -t links 'links' _links
+						_describe -t links 'links' _links _links $( [[ $cmd == (edit|cat) ]] && print -- -P@ )
```

Comment:
Please adjust the indentation.

Also, `_links` is duplicated?

Reply:
> Please adjust the indentation.

Whoops, will do.

> Also, _links is duplicated?

That's intentional. Even though they are flags, _describe only takes completion options in that final argument position so I need something to pass in the 3rd argument position. Passing _links twice allows that without changing the completions.